src/taskFunctions.js

```diff
@@ -7,19 +7,30 @@ export function getTasks() {
 
 export function setTasks(newTasks) {
   tasks = newTasks;
+  // eslint-disable-next-line no-use-before-define
+  saveTasksToLocalStorage(tasks);
+}
+
+export function saveTasksToLocalStorage(tasks) {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
```

Comment:
- [ ] As stated by the previous reviewer, kindly consider after saving todos in local storage and refresh the page the todos displayed from local storage.